### DISCUSSION TITLE
change volume by hovering over a specific area.

### DISCUSSION THREAD

**Discussion by zedd-0** [File-New-Project/EarTrumpet#1459] [Ideas]
I am familiar with the Volume² function where the volume can be changed when the mouse hovers over the taskbar and can be adjusted using the mouse wheel.

It was also possible to create a custom-defined area, for example, the top-left corner, 60x60 pixels.

However, I would try to use relative values instead of fixed pixel coordinates to ensure it works smoothly with DPI scaling.

For me personally, this is a killer feature! 😄
I'd like to avoid using 10,000 tools just for optimizing volume features as well. 😉

thanks for reading.

**Comment by riverar** [comment]
@zedd-0 Thanks for the idea; are you aware this is already implemented for the system tray icon at least?

**Reply by zedd-0** [reply]
I forgot to mention that should also works in full screen.
Even with a multi-monitor setup. (more arias/taskbars)
It's just hard to hit such a small icon on my monitor.
My feature request could be used blindly and intuitively.
Just move the mouse pointer down or into a corner and scroll the mouse wheel.
This would even work with turned off/slow monitors. :D

**Reply by riverar** [reply]
We also have a feature where we accept the wheel input anywhere, when the flyout is open. So triggering the flyout (e.g. CTRL+~) then using the wheel is pretty fast/easy. Just making sure you're aware of the current options.

**Reply by zedd-0** [reply]
I have tested it.

It's not as intuitive as I imagined.
My thought is, if I define these areas,
they are always active, and I don't need to press another key combination with a second hand.

This is a bit far-fetched, but what do people with only one arm do? XD
